ext/date/tests/DateTime_createFromInterface.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Tests for DateTime::createFromInterface
+--INI--
+date.timezone=Europe/London
+--FILE--
+<?php
+$current = "2014-03-02 16:24:08";
```

Comment:
Please also add a test for a `timezone_type=1` date/time object: (Using `2019-12-16 15:06:46 CET`) and a `timezone_type=2` one (Using for example `2019-12-16 15:08:20 +0100` string).